scenarios/basics/inspector/SPECIFICATION.md

```diff
@@ -18,72 +23,106 @@ This document contains a draft proposal for an *Amazon Inspector Basics Scenario
 * [ListCoverage](https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListCoverage.html)
 * [Disable](https://docs.aws.amazon.com/inspector/v2/APIReference/API_Disable.html)
 
-## Proposed example structure
+## Hello Amazon Inspector
 
-The output below demonstrates how this example would run for the customer. It includes a Hello service example (included for all services), and the scenario description. The scenario code would also be presented as Action snippets, with a code snippet for each SDK action.
+The Hello example is intended for users not familiar with this service to easily get up and running. It sets up the Inspector service client, checks the current account status for Inspector and displays available scan types.
 
-### Hello
+## Scenario
 
-The Hello example is a separate runnable example. - Set up the Inspector service client - Check the current account status for Inspector - Display available scan types and regions
+This scenario demonstrates the basic usage of **Amazon Inspector** using a Java program. It focuses on checking account status, enabling Inspector, listing findings, reviewing coverage, and managing filters.
 
-## Scenario
+---
 
-#### Setup
+### Setup
 
-* Enable Amazon Inspector for the account
-* Verify Inspector is successfully activated
-* Display account status and enabled scan types
+* Check Amazon Inspector account status
+* Enable Inspector for available resource types (if not already enabled)
+* Display account status summary
 
-#### Coverage Assessment
+---
 
-* List coverage statistics for EC2 instances, ECR repositories, and Lambda functions
-* Display resource coverage details
-* Show scanning status for different resource types
+### Coverage Assessment
 
-#### Findings Management
+* List coverage details for scanned resources
+* Display overall coverage statistics
+* Review scan status for resources (general overview)
+
+---
+
+### Findings Management
 
 * List security findings across all resource types
-* Filter findings by severity level (CRITICAL, HIGH, MEDIUM, LOW)
-* Retrieve detailed information for specific findings
+* Create an example filter to suppress low-severity findings
+* List existing filters
+
+---
 
-#### Vulnerability Analysis
+### Usage and Costs
 
-* Display vulnerability details including CVE information
-* Show affected resources and remediation guidance
-* Filter findings by resource type (EC2, ECR, Lambda)
+* Check usage totals and metrics for Inspector
+* Review coverage statistics
 
-#### Cleanup
+---
 
-* Optionally disable Inspector scanning (with user confirmation)
-* Display final account status
+### Notes
+
+* The program **does not retrieve detailed vulnerability (CVE) information**.
+* Resource-specific filtering (e.g., EC2, ECR, Lambda) is **not implemented**.
+* Cleanup (disabling Inspector) is **not performed**.
+
+---
+
```

Comment:
Where is the Cleanup section? Are we leaving everything in the same state as how we started?